content/installation/setup/installation/UsingZooKeeperProperties.md

```diff
@@ -0,0 +1,55 @@
+<!--
+title: "Configuring Contrast To Use ZooKeeper"
+description: "Instructions for configuring TeamServer to use ZooKeeper as the single source of truth"
+tags: "installation EOP distributed configuration ZooKeeper"
+-->
+
+## About ZooKeeper and Contrast
+This guide is for enterprise-on-premises (EOP) administrators who are looking to move to a distributed setup of Contrast and want to use ZooKeeper as their configuration management. 
+
+> **Note:** While the following steps guide you through the setup and configuration of additional software, keep in mind that you're responsible for the monitoring, backups and durability of ZooKeeper. 
```

Comment:
backup (singular) makes more sense to me here (backup of Zookeeper vs backups of Zookeeper)